lib/smart_answer_flows/uk-benefits-abroad.rb

```diff
@@ -601,14 +592,151 @@ def define
         end
       end
 
+      # Going abroad
+      radio :worked_in_eea_or_switzerland? do
+        option :before_jan_2021
+        option :after_jan_2021
+        option :no
+
+        next_node do |response|
+          case response
+          when "before_jan_2021"
+            if calculator.benefit == "jsa"
+              outcome :jsa_eea_going_abroad_maybe_outcome
+            elsif calculator.benefit == "winter_fuel_payment"
+              outcome :wfp_going_abroad_eea_maybe_outcome
+            elsif calculator.benefit == "esa"
+              if calculator.going_abroad
+                outcome :esa_going_abroad_eea_outcome
+              else
+                outcome :esa_already_abroad_eea_outcome
+              end
```

Comment:
This (and several examples below) might benefit from being ternary statements:

```ruby
elsif calculator.benefit == "esa"
  outcome calculator.going_abroad ? : esa_going_abroad_eea_outcome : : esa_already_abroad_eea_outcome
```

Helps avoid the code from getting too 'nested' :)